lib/page/detail_page.dart

```diff
@@ -254,9 +250,40 @@ List<Row> getCategoriesList(List<PlaceCategory>? categories) {
       .toList();
 }
 
+// null safety
```

Comment:
is this comment needed?

Reply:
intentional put so it acts like an ''fence area'